packages/api/src/commands/install.ts

```diff
@@ -120,14 +120,12 @@ cmd
       .catch(err => {
         // @todo cleanup installed files
         spinner.fail(spinner.text);
-        logger(err.message, true);
-        process.exit(1);
+        throw err;
```

Comment:
you'll see this kind of refactor a lot — i consolidated our error logging + `process.exit(1)` in a `.catch()` in `packages/api/src/bin.ts` so we can more easily test this.